src/test/java/io/vinyldns/java/VinylDNSClientTest.java

```diff
@@ -147,14 +183,122 @@ public void getZoneFailure404() {
         get(urlEqualTo("/zones/" + zoneId))
             .willReturn(aResponse().withStatus(404).withBody("not found")));
 
-    VinylDNSResponse<GetZoneResponse> vinylDNSResponse = client.getZone(new GetZoneRequest(zoneId));
+    VinylDNSResponse<GetZoneResponse> vinylDNSResponse = client.getZone(new ZoneRequest(zoneId));
 
     assertTrue(vinylDNSResponse instanceof ResponseMarker.Failure);
     assertEquals(vinylDNSResponse.getStatusCode(), 404);
     assertEquals(vinylDNSResponse.getMessageBody(), "not found");
     assertNull(vinylDNSResponse.getValue());
   }
 
+  @Test
+  public void updateZoneSucess() {
+    Zone newZone = new Zone(
```

Comment:
We need to stringify both the requests and responses for these tests using `client.gson.toJson(...);`